test/DaedalusCompiler.Tests/AssemblyBuilderTest.cs

```diff
@@ -0,0 +1,181 @@
+﻿using System.Collections.Generic;
```

Comment:
It's integration test,
maybe name file and class `ParserAndAssemblyBuilderIntegrationTest.cs`